frontend/simple/js/backend/hapi.js

```diff
@@ -95,6 +96,12 @@ export class HapiBackend extends Backend {
     store.dispatch('saveSettings')
     return res
   }
+  async latestHash (contractId: string) {
+    return fetch(`${process.env.API_URL}/latestHash/${contractId}`)
+  }
+  async eventStream (contractId: string, since: string) {
+    return fetch(`${process.env.API_URL}/events/${contractId}/${since}`)
+  }
```

Comment:
OK, this mix-and-matching of superagent and fetch is telling me that we need to return the value, not the implementation-specific result object.

E.g. `recentHash` should return the recent hash (`Promise<string>`) or throw on error.
`eventStream` should, for now, be renamed to `eventsSince` and return an array of objects. Plus a `TODO:` comment reminder to implement `eventStream` in the future, returning a `stream.Transform` (or whatever it is in the browser) to efficiently go through the results one-by-one so that they're not stored in memory.